### PR TITLE
Adding new cppe orb for running sast on terraform

### PR DESCRIPTION
**Pull request by ovo-dsingh**

### CODE REVIEW

File: sast-terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# SAST Terraform CircleCI orb
+
+This orb can be used to run the static code analysis tools via the CLI against a target directory containing Terraform infrastructure-as-code.
+
+## Commands
+### checkov-static_code_analysis
```
Comment (chris-brindley) [inline sast-terraform/README.md:6]:
```suggestion
### checkov_static_code_analysis
```

File: sast-terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+executors:
+  python:
+    docker:
+    - image: circleci/python:3.9
+
+jobs:
+  run-checkov:
+    executor: python
+    steps:
+    - sast-terraform/checkov-static-code-analysis:
```
Comment (chris-brindley) [inline sast-terraform/README.md:31]: Should the job name here should also use underscores?

File: sast-terraform/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# SAST Terraform CircleCI orb
+
+This orb can be used to run the static code analysis tools via the CLI against a target directory containing Terraform infrastructure-as-code.
```
Comment (chris-brindley) [inline sast-terraform/README.md:3]: Maybe worth also adding a little bit about the baseline file and how to generate it to ease adoption for teams?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -19,3 +19,4 @@
 /argocd/                  @ovotech/metering-industry-abstraction-cluster-admins
 /comms-commands/          @ovotech/comms
 /ipa-deploy/              @ovotech/ipa
+/sast-terraform/          @ovotech/consumer-products-production-engineering
```
Comment (chris-brindley) [inline .github/CODEOWNERS:22]: Our team is lacking write access to the repo which is causing this to display as an error

File: sast-terraform/README.md (hunk 2 above)
Comment (ovo-dsingh) [reply sast-terraform/README.md]: Good catch!

File: .github/CODEOWNERS (hunk 4 above)
Comment (ovo-dsingh) [reply .github/CODEOWNERS:22]: Mentioned that the team needs this access in the PR description

**Review by chris-brindley — APPROVED** [review]
🚀

**Pull request merged**